snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -113,6 +113,12 @@ init_config:
   #   netapp:
   #     definition_file: netapp.yaml
 
+  ## @param collect_device_metadata - boolean - optional - default: false
+  ## Enable device metadata collection
+  ## Only available using corecheck SNMP integration with `loader: core` config.
+  #
+  # collect_device_metadata: true
```

Comment:
Default value should be false for now.

Reply:
@gmarabout The default is already false:
```
  ## @param collect_device_metadata - boolean - optional - default: false<br class="Apple-interchange-newline">
```

The the line 120:
```
  # collect_device_metadata: true
```

Is a "suggested" value, so that, when the user uncomment the line, it's set to `true`. If user uncomment this line, they probably want it to be `true`.